code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -7,19 +7,22 @@ GLOBAL_DATUM_INIT(temp_reagents_holder, /obj, new)
 	var/atom/my_atom = null
 
 /datum/reagents/New(maximum_volume = 120, atom/my_atom)
+	SHOULD_CALL_PARENT(TRUE)
 	if(!istype(my_atom))
 		CRASH("Invalid reagents holder: [log_info_line(my_atom)]")
 	..()
 	if(!GLOB.chemical_reagents_list)
 		build_chemical_reagent_list()
 	src.my_atom = my_atom
 	src.maximum_volume = maximum_volume
+	GLOB.reagents_datums += src
```

Comment:
What is this list for?

Reply:
It seems to be used in 294's new way of finding the reagent holders.